pkg/factory/factory_test.go

```diff
@@ -47,6 +47,28 @@ var _ = Describe("Factory", func() {
 			})
 		})
 	})
+	DescribeTable("getting allocator",
+		func(policy string, shouldSucceed bool, expected reflect.Type) {
+			f := factory.NewResourceFactory("fake", "fake", true)
+			a, e := f.GetAllocator(policy)
```

Comment:
Could we use different var names here?
a -> allocator ?
e -> error  ?

It would make it easier to read.

Reply:
Renamed variables.